ACP2E-346: [Documentation] Add final minimum(maximum) price calculation formula into number of documents (#1635)
magento/merchdocs: src/marketing/price-rules-catalog-create.md

   |Adjust final price to discount value|Sets the final price to a fixed, discounted amount. For example: Enter 20 in Discount Amount for a final price of $20.00.|

    {:.bs-callout-info}

>     _Regular price_ refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. _Final price_ refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)

Reviewer: ```suggestion
    _Regular price_ refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. _Final price_ refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`
```

Author: Committed

### Actions

> |Apply|Determines the type of calculation that is applied to the purchase. Options: <br/>**Apply as percentage of original** - Discounts item by subtracting a percentage of the regular price. <br/>**Apply as fixed amount** - Discounts item by subtracting a fixed amount from the regular price. <br/>**Adjust final price to this percentage** - Adjusts the final price by a percentage of the regular price. <br/>**Adjust final price to discount value** - Sets the final price to a fixed, discounted amount. <br/><br/>**_Note:_** Regular price refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. Final price refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, by the formula: <br/>Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)|

Reviewer: ```suggestion
|Apply|Determines the type of calculation that is applied to the purchase. Options: <br/>**Apply as percentage of original** - Discounts item by subtracting a percentage of the regular price. <br/>**Apply as fixed amount** - Discounts item by subtracting a fixed amount from the regular price. <br/>**Adjust final price to this percentage** - Adjusts the final price by a percentage of the regular price. <br/>**Adjust final price to discount value** - Sets the final price to a fixed, discounted amount. <br/><br/>**_Note:_** Regular price refers to the base product price without any advanced pricing (special/tier/group) or promotional discounts. Final price refers to the discounted price that appears in the shopping cart. <br/>The **_final_** product price is calculated as the **_minimum_** relevant price, using the following formula: <br/>`Final Price=Min(Regular(Base) Price, Group(Tier) Price, Special Price, Catalog Price Rule) + Sum(Min Price per each required custom option)`|
```
